application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/WolframAlphaCommand.java

```diff
@@ -0,0 +1,374 @@
+                                .formatted(++filesAttached))
+                            .toFile());
+                        if (message == null) {
+                            message = channel.sendFile(WolfCommandUtils.imageToBytes(combinedImage),
+                                    "result%d.png".formatted(++filesAttached));
+                        } else {
+                            message.addFile(WolfCommandUtils.imageToBytes(combinedImage),
+                                    "result%d.png".formatted(++filesAttached));
+                        }
+                        /*
+                         * messages.add(channel.sendFile(WolfCommandUtils.imageToBytes(combinedImage
+                         * ), name));
+                         */
+                        /*
+                         * bytes.add(WolfCommandUtils.imageToBytes(combinedImage));//
+                         * filesAttached++;
+                         */
+                        embeds.add(embedBuilder
+                            .setImage("attachment://result%d.png".formatted(filesAttached))
+                            .build());
+                    }
+                    resultHeight += readImage.getHeight();
+                    images.add(readImage);
+                    logger.info(
+                            "Max Width {} Result Height {} Current Width {} Current Height {} Images {}",
+                            maxWidth, resultHeight, readImage.getWidth(), readImage.getHeight(),
+                            filesAttached);
+                } catch (IOException e) {
+                    event.reply("Unable to generate message based on the WolframAlpha response")
+                        .setEphemeral(true)
+                        .queue();
+                    logger.error("Failed to read image {} from the WolframAlpha response", image,
+                            e);
+                    return Optional.empty();
+                }
+            }
+            logger.info("Exiting pod number {}", i);
+        }
+        /*
+         * for (int i = bytes.size() - 1; i >= 0; i--) { try { ImageIO.write(ImageIO.read(new
+         * ByteArrayInputStream(bytes.get(i))), "png", Path .of(
+         * "C:\\Users\\Abc\\IdeaProjects\\TJ-Bot-baseRepo\\application\\src\\main\\java\\org\\togetherjava\\tjbot\\commands\\mathcommands\\wolframalpha\\sentImage%d.png"
+         * .formatted(i)) .toFile()); } catch (IOException e) { e.printStackTrace(); } action =
+         * action.addFile(bytes.get(i), names.get(i)); }
+         */
+        messages.add(message.setEmbeds(embeds));
+        return Optional.of(messages);
+    }
+
+    private int getWidth(String header) {
+        BufferedImage image = new BufferedImage(100, 100, 6);
+        Graphics g = image.getGraphics();
+        g.setFont(WOLFRAM_ALPHA_FONT);
+        return g.getFontMetrics().stringWidth(header);
```

Comment:
```suggestion
        BufferedImage image = new BufferedImage(100, 100, 6);
        
        Graphics g = image.getGraphics();
        g.setFont(WOLFRAM_ALPHA_FONT);
        
        return g.getFontMetrics().stringWidth(header);
```
And I still feel like there should come more empty lines in the code, makes it a lot more readable, especially in the really really long methods.

Reply:
This method is literally 4 lines. Adding more lines does not help much imo

Reply:
It was an example, because there's many a lot bigger methods that do need this.

And I'd say it even helps in this method, makes it more consistent imo.
But your decision whenever you'll change it in this method or not.